CONFIGURING-DETECTORS.md

```diff
@@ -0,0 +1,103 @@
+
+## Why would I want to configure detectors?
+
+Configuring detectors for the edge endpoint allows you to provide finer-grained control over their behavior on the edge. By configuring a detector, you can:
+* Have the edge endpoint automatically create an inference pod for a detector, ensuring it's ready to serve requests without having to first submit a query to it
+* Configure a detector to always return the edge model's answer, regardless of the confidence
+* Configure a detector to never escalate queries to the cloud
+* Specify a maximum frequency for cloud escalations
+
+## How do I configure a detector?
+
+Detector configurations are specified in [edge-config.yaml](configs/edge-config.yaml). The file has three sections: `global_config`, `edge_inference_configs`, and `detectors`. 
+
+NOTE: After modifying the config file, you'll have to re-run [setup-ee.sh](deploy/bin/setup-ee.sh) for your changes to be reflected.
+
+### `global_config`
+
+The global config contains parameters that affect the overall behavior of the edge endpoint. 
+
+#### `refresh_rate`
+
+`refresh_rate` is a float that defines how often the edge endpoint will attempt to fetch updated ML models (in seconds). If you expect a detector to frequently have a better model available, such as if you are labeling many image queries for a new detector, you can reduce this to ensure that the improved models will quickly be fetched and deployed. In practice, you likely won't want this to be lower than ~30 seconds due to the time it takes to train and fetch new models. If not specified, the default is 60 seconds.
+
+### `edge_inference_configs`
+
+Edge inference configs are 'templates' that define the behavior of a detector on the edge. Each detector you configure will be assigned one of these templates. There are some pre-defined configs that represent the main ways you might want to configure a detector. However, you can edit these and also create your own as you wish.
+
+#### Structure of an edge_inference_config
+
+For each edge inference config, you can configure various parameters. For a complete description of each, see [Appendix: Edge Inference Parameters](#appendix-edge-inference-parameters) below.
+
+#### Pre-defined edge inference configs
+
+There are a few pre-defined configs that represent the main ways you might want to configure a detector. 
+
+##### `default`
+```
+default: # Return the edge model's prediction if sufficiently confident; otherwise, escalate to the cloud.
+    enabled: true
+    always_return_edge_prediction: false
+    disable_cloud_escalation: false
+```
+This is the default behavior for a detector on the edge and is likely what you'll want to use most of the time, unless you have a specific reason to use a different configuration.
+
+##### `edge-answers-with-escalation`
+```
+edge-answers-with-escalation: # Always return the edge model's predictions, but still escalate to cloud if unconfident.
```

Comment:
I'm confused the example section of these. If edge-answers-with-escalation is a combination of those parameters, are you saying we have to still include them in the config file? Couldn't we just specify - edge-answers-with-escalation?